drafts/inlinemacros.md

```diff
@@ -25,9 +26,7 @@ We introduce a new reserved word: `inline`. `inline` can be used as a modifier f
 
 The previous rule of regarding a `final val` with no explicit type as
 a compile-time constant (inherited from Java) is dropped. Instead we
-write such `val`s now as `inline`.  Inline values can be defined
-anywhere, but inline methods must be members of a class, trait or
-object.
+write such `val`s now as `inline`.
```

Comment:
Agreed. The previous restriction was there because it made it easier to talk about the prefix of an inline method call. Since we don't do prefixes anymore in the inline world, it can go.